### PR TITLE
[CUMULUS-2585] Add execution events snapshot to executions page

### PR DESCRIPTION
**Pull request by dopeters**
**Summary:**

Addresses [CUMULUS-2585: Dashboard: Failed Execution Steps Snapshot & Reference Link](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2585)

## Changes

* Adds Progress columns to Execution Overview page
* Adds Failed Event Snapshot to granule executions list
* Updated action/reducer to prevent incorrect data

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Adhoc testing
- [x] Integration tests

### CODE REVIEW

**Comment by jjmccoy** [comment]
UI/UX Review: This review will be regarding the interface and interactions. We will need another review for code/logic.

**Comment by jjmccoy** [comment]
@dopeters Amazing job with this challenging ticket. UI/UX LGTG!

**Comment by jennyhliu** [comment]
Why is the package-lock.json updated when there is no package change?

File: app/src/js/components/Executions/executions-list.js [hunk 1]
```diff
@@ -28,9 +29,11 @@ const ExecutionsList = ({
   match,
   executions
 }) => {
-  const { list } = executions || { list: { meta: { count: 0 }, results: [] } };
   const { params } = match || {};
   const { collectionId, granuleId } = params;
+  const { map } = executions || {};
+  const list = map[granuleId] || initialState.list;
```
Comment (jennyhliu) [inline app/src/js/components/Executions/executions-list.js:35]: Does the `map[granuleId]` have the same structure as initialState.list?

Comment (jennyhliu) [reply app/src/js/components/Executions/executions-list.js:35]: I'm confused the `list` with `executions.list`.  Can we rename it to something like `granuleExecutionsList`?

**Comment by dopeters** [comment]
@jennyhliu setState loop should be resolved if you're able to give this another look.

**Review by jennyhliu — APPROVED** [review]
Great job!

**Pull request merged**